src/Idris/Package.idr

```diff
@@ -711,6 +711,7 @@ partitionOpts opts = foldr pOptUpdate (MkPFR [] [] False) opts
     optType (OutputDir f)          = POpt
     optType WarningsAsErrors       = POpt
     optType HashesInsteadOfModTime = POpt
+    optType Profile          = POpt
```

Comment:
It would be good to keep the `=` alignment.

Reply:
Sorry, I wasn't able to figure out how to change this pull request, and I accidentaly created a new one, #1576 , which has the spacing fixed, so I'm closing this.

Reply:
No worries uwu